dispatch/static/manager/src/js/components/ArticleEditor/ArticleSidebar.js

```diff
@@ -44,82 +45,117 @@ const tabData = [
-        </TabPanel>
-      </Tabs>
-    </div>
-  )
+  renderSideBar() {
+    return (
+      <div style={{height: '100%'}}>
+        <Tabs>
+          <TabList className='c-article-sidebar__tablist'>
+            {tabData.map((data, index) => (this.renderTab(data, index, Object.keys(this.props.errors))))}
+          </TabList>
+  
+          <TabPanel className='c-article-sidebar__panel'>
+            <BasicFieldsTab
+              update={this.props.update}
+              section={this.props.article.section}
+              authors={this.props.article.authors || []}
+              tags={this.props.article.tags || []}
+              topic={this.props.article.topic}
+              slug={this.props.article.slug}
+              snippet={this.props.article.snippet}
+              errors={this.props.errors} />
+          </TabPanel>
+  
+          <TabPanel className='c-article-sidebar__panel'>
+            <FeaturedImageTab
+              update={this.props.update}
+              featured_image={this.props.article.featured_image}
+              entities={this.props.entities} />
+          </TabPanel>
+  
+          {/* uncomment when featured videos are ready */}
+          {/* <TabPanel className='c-article-sidebar__panel'>
+            <FeaturedVideoTab
+              update={this.props.update}
+              featured_video={this.props.article.featured_video}
+              entities={this.props.entities} />
+          </TabPanel> */}
+  
+          <TabPanel className='c-article-sidebar__panel'>
+            <DeliveryTab
+              update={this.props.update}
+              importance={this.props.article.importance}
+              reading_time={this.props.article.reading_time}
+              integrations={this.props.article.integrations}
+              is_breaking={this.props.article.is_breaking}
+              breaking_timeout={this.props.article.breaking_timeout}
+              availableIntegrations={this.props.integrations} />
+          </TabPanel>
+  
+          <TabPanel className='c-article-sidebar__panel'>
+            <TemplateTab
+              update={this.props.update}
+              template={this.props.article.template || 'default'}
+              data={this.props.article.template_data || {}}
+              errors={this.props.errors} />
+          </TabPanel>
+  
+          <TabPanel className='c-article-sidebar__panel'>
+            <SEOTab
+              update={this.props.update}
+              headline={this.props.article.headline}
+              slug={this.props.article.slug}
+              seo_keyword={this.props.article.seo_keyword || ''}
+              seo_description={this.props.article.seo_description || ''} />
+          </TabPanel>
+        </Tabs>
+      </div>
+    )
+  }
+  
+  render () {
+    const isDesktop = window.document.body.clientWidth > desktopSize 
+    const open = isDesktop ? '' : (this.state.isOpen ? 'open': 'closed')
+    const sliderStyle = {
```

Comment:
Would we be able to move this to a style sheet instead?

Reply:
woops